### PR TITLE
[ENHANCEMENT] Improved address handling; Cleanups

### PR DESCRIPTION
**Pull request by turbo-ele**

### CODE REVIEW

File: Cart/CartManager.php [hunk 1]
```diff
@@ -428,12 +426,12 @@ private function findCartBySessionId()
     /**
      * Finds cart by locale and user
      *
-     * @param $locale
-     * @param $user
+     * @param string $locale
+     * @param UserInterface $user
      *
      * @return array|null
```
Comment (christianbader) [inline Cart/CartManager.php:432]: array? or Cart?

File: Order/OrderManager.php [hunk 2]
```diff
@@ -397,12 +407,14 @@ protected function createMediaForItems($items, $locale)
     }
 
     /**
-     * returns contact data as an array. either by provided address or contact
+     * Returns contact data as an array. either by provided address or contact
```
Comment (christianbader) [inline Order/OrderManager.php]: Either

File: Order/OrderManager.php [hunk 3]
```diff
@@ -397,12 +407,14 @@ protected function createMediaForItems($items, $locale)
...
      *
      * @param $addressData
```
Comment (christianbader) [inline Order/OrderManager.php]: type?

File: Order/OrderManager.php [hunk 4]
```diff
@@ -436,8 +448,10 @@ public function getContactData($addressData, $contact)
     }
 
     /**
-     * deletes an order
+     * Deletes an order
+     *
      * @param $id
```
Comment (christianbader) [inline Order/OrderManager.php]: int

File: Order/OrderManager.php [hunk 5]
```diff
@@ -600,7 +634,9 @@ public function getFieldDescriptors($locale)
 
     /**
      * returns a specific field descriptor by key
+     *
      * @param $key
```
Comment (christianbader) [inline Order/OrderManager.php]: type

File: Order/OrderManager.php [hunk 6]
```diff
@@ -610,8 +646,10 @@ public function getFieldDescriptor($key)
 
     /**
      * Finds an order by id and locale
+     *
      * @param $id
```
Comment (christianbader) [inline Order/OrderManager.php]: type

File: Order/OrderManager.php [hunk 7]
```diff
@@ -824,7 +866,8 @@ private function initializeFieldDescriptors($locale)
     }
 
     /**
-     * check if necessary data is set
+     * Check if necessary data is set
+     *
      * @param $data
```
Comment (christianbader) [inline Order/OrderManager.php]: type

File: Cart/CartManager.php (hunk 1 above)
Comment (turbo-ele) [inline Cart/CartManager.php:432]: array

**Comment by christianbader** [comment]
tests? else ack

**Comment by turbo-ele** [comment]
test will work again, after PR in SalesCore is merged

**Pull request merged**